test/default_test.rb

```diff
@@ -1,34 +1,75 @@
 require "test_helper"
 
 class DefaultTest < MiniTest::Spec
-  Song = Struct.new(:id, :title)
+  TIMESTAMP = Time.now
+
+  Composer = Struct.new(:id, :name, :keywords)
+  Song = Struct.new(:id, :title, :album, :composers, :created_at, :created_by)
 
   representer! do
     property :id
     property :title, default: "Huber Breeze" #->(options) { options[:default] }
+    property :album, default: ->(represented:, **) { represented.album || "Spring" }
+    collection :composers, instance: ->(*) { Composer.new } do
```

Comment:
It would be very helpful if this _collection_ also showed how to use a `default:`.
BTW, is there a benefit of using `instance: ->(*) { Composer.new }` instead of `class: Composer`? The tests seem to pass either way.